src/config/services/servicesConfig.go

```diff
@@ -103,5 +113,11 @@ func DecodeConfig(cfg config.Provider) (c Config, err error) {
 		c.AssetLinks.Fingerprint = cfg.GetString(assetLinksFingerprintKey)
 	}
 
+	if c.AppLinks.Prefix == "" {
+		c.AppLinks.Prefix = cfg.GetString(appLinkPrefixKey)
+		c.AppLinks.Bundle = cfg.GetString(appLinkBundleKey)
+		c.AppLinks.AppID = cfg.GetString(appLinkPrefixKey) + "." + cfg.GetString(appLinkBundleKey)
```

Comment:
@felicianotech Could I do this in the tpl file instead? If so, I can simplify the struct and remove this line.

Reply:
I was able to figure this out.

Reply:
This is a Go source file (hence the `.go` extension). Anything that any other Go file can use of Go syntax this can. The limitations apply when you get to files with other extensions.